command/show.go

```diff
@@ -186,8 +186,8 @@ Usage: terraform show [options] [path]
 Options:
 
   -no-color           If specified, output won't contain any color.
-  -json				  If specified, output the Terraform plan in a machine-
-						readable form. Only available for plan files.
+  -json				  If specified, output the Terraform plan or state in
```

Comment:
It looks like there are some tabs lurking in here, causing the alignment to be off when rendered with a different tab stop setting.

Reply:
AH

I fixed that in the other one but wasn't looking for it here.

Reply:
...oh hey, this entire .go file uses hardtabs, except for the usage text. Kind of wild, but now that I'm looking at the others it seems like that's the convention. 

```suggestion
  -json               If specified, output the Terraform plan or state in
```

Reply:
Yeah, `.go` files always use tabs for intendation per the `go fmt` tool, but our usage strings do not because we can't predict the tab stop settings for the terminal.